ElectronClient/gui/NoteText.jsx

```diff
@@ -1461,7 +1461,12 @@ class NoteTextComponent extends React.Component {
 					let start = selectedStrings[i].search(/[^\s]/);
 					let end = selectedStrings[i].search(/[^\s](?=[\s]*$)/);
 					newBody += selectedStrings[i].substr(0, start) + string1 + selectedStrings[i].substr(start, end - start + 1) + string2 + selectedStrings[i].substr(end + 1);
-					if (this.state.note.body.substr(selection.end) === '') newBody = newBody.trim();
+
+					// correcting selction offset
+					var startCursorPos = this.textOffsetToCursorPosition(selection.start+start, newBody);
+					var endCursorPos = this.textOffsetToCursorPosition(selection.start+end+1,newBody);
```

Comment:
var => const

Reply:
we use these variables in other blocks too, what do you prefer
 

-  define them with `let` outside the block

- let these variables be defined with `var`